### DISCUSSION TITLE
What defines a good candidate for code ownership by the foundation?

### DISCUSSION THREAD

**Discussion by dalemyers** [MobileNativeFoundation/discussions#21] [General]
Already we have 4 projects owned by the foundation:

* https://github.com/MobileNativeFoundation/Kronos
* https://github.com/MobileNativeFoundation/index-import
* https://github.com/MobileNativeFoundation/set-simulator-location
* https://github.com/Flank/flank (moving shortly I presume)

These vary in what they do, size, complexity, etc. 

What defines a good project for the foundation? Anything that is potentially useful would be a very broad umbrella, and would bring in projects that are relatively unused, unsupported, etc. Focusing on completeness and users could restrict some smaller projects coming to light and being discovered by the community. 

So, how do we decide what projects are suitable for ownership by the foundation?

**Comment by dalemyers** [comment]
I've created a proposal which outlines some of the criteria I think would be useful to help make this decision: https://github.com/MobileNativeFoundation/discussions/discussions/46

**Reply by keith** [reply]
Thanks for writing this up! I'm going to take a look at it in more detail soon, I'm also hoping this is something we can bring to the first committee meeting to discuss
